templates/node/src/Payload.ts.twig

```diff
@@ -0,0 +1,58 @@
+import { readFileSync } from "fs";
+import { File } from "node-fetch-native-with-agent";
+import { basename } from "path";
+
+export class Payload extends File {
```

Comment:
Do we need to `extend File` here?

Reply:
It was added because `upload` method in this example expects a `File` object
`response = await general.upload('string', 123, ['string in array'], Payload.fromFile(__dirname + '/../../resources/large_file.mp4', 'large_file.mp4'));`